ProfileManager.js

```diff
@@ -153,27 +150,20 @@ export default class ProfileManager {
     }
 
     promise = (async () => {
-      const capabilityCache = this._cache.get(capabilityCacheKey);
+      const capabilityCache = this._cacheContainer.get(capabilityCacheKey);
       let zcap = capabilityCache.get(capabilityKey);
       if(!zcap) {
-        const agentSigner = await this._getInvocationSigner(
-          {profileAgentId: profileAgent.id, capabilityInvoker: 'agent'});
-        if(id === PROFILE_INVOCATION_ZCAP_REFERENCE_ID) {
-          assert.nonEmptyString(profileId, 'profileId');
-          id = _getProfileInvocationZcapKeyReferenceId({
-            zcaps: profileAgent.zcaps,
-            profileId
-          });
-        }
+        const agentSigner = await this.getAgentSigner(
+          {profileAgentId: profileAgent.id, useEphemeralSigner: false});
         const originalZcap = profileAgent.zcaps[id];
         if(!originalZcap) {
           const {id: profileAgentId} = profileAgent;
           throw new Error(
             `The agent "${profileAgentId}" does not have the zcap: "${id}"`);
         }
-        if(capabilityInvoker === 'local') {
-          const localSigner = await this._getInvocationSigner(
-            {capabilityInvoker: 'local'});
+        if(useEphemeralSigner) {
+          const localSigner = await this.getAgentSigner(
```

Comment:
I suggest we don't use `local` anywhere.

Reply:
Leftover variable name from the previous implementation. Fixed.